edenai_apis/features/ocr/ocr_interface.py

```
@@ -165,6 +165,18 @@ def ocr__identity_parser(
         """
         raise NotImplementedError
 
+    @abstractmethod
+    async def ocr__aidentity_parser(
+        self, file: str, file_url: str = "", model: str = None, **kwargs
```

Ruff (0.14.4)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)